efi/signature/signature_list.go

```diff
@@ -285,15 +287,21 @@ func ReadSignatureList(f io.Reader) (*SignatureList, error) {
 	switch sig {
 	case "X509":
 		if s.HeaderSize != 0 {
-			log.Fatalf("Unexpected HeaderSize for x509 cert. Should be 0!")
+			msg := "Unexpected HeaderSize for x509 cert. Should be 0!"
+			log.Printf(msg)
```

Comment:
Lets not print the messages.

Reply:
I did print it to keep the same behavior as before, but rather return the error indeed :)